src/client/core/utils/array.js

```diff
@@ -66,3 +66,18 @@ export function remove (arr, item) {
     if (index > -1)
         nativeSplice.call(arr, index, 1);
 }
+
+export function equals (arr1, arr2) {
+    if (!arr2)
```

Comment:
There is no case. `domUtils.getParents` returns empty array, if argument was top element